service/build.gradle

```diff
@@ -177,7 +177,7 @@ repositories {
 // it is important to specify logback classic and core packages explicitly as libraries like spring boot
 // enforces it's own (older) version which is not recommended.
 def versions = [
-  reformLogging: '5.0.1',
+  reformLogging: '5.1.0',
```

Comment:
I think thanks for that change, we can now remove the netty.io exclusion which I have added in the owasp config.